megamek/src/megamek/common/Crew.java

```diff
@@ -1354,10 +1385,64 @@ public boolean doConsoleRoleSwap() {
         return false;
     }
 
+    //region extraData
```

Comment:
Are all of these overloads used/necessary?

Reply:
If you mean the region tags, they make it far easier to parse through files in the IDE... so I would say yes.
If you mean the methods, probably not. I made a bunch just in case I need them, but wanted some advice on my implementation (to ensure I'm not doing something crazy) before I went and did the full implementation. I'll remove anything that I find useless following my implementation, or potentially leave in with documentation on how to use them.

Reply:
I'd prefer removing anything unused.